apps/synapse-portal-framework/src/components/DetailsPage/DetailsPageContentLayout.tsx

```diff
@@ -7,19 +8,6 @@ import {
 
 export type DetailsPageContentType = DetailsPageSectionLayoutType[]
 
-export const handleMenuClick = (id: string) => {
```

Comment:
moved into component to use `useLocation`/`useNavigate` hooks instead of `window`